fcm/atmosphere/_mars_atm_api.py

```diff
@@ -32,41 +32,59 @@ def martian_atmosphere_api(latitude, longitude, timestamp):
         
     timestamp : Union[datetime.date, datetime.datetime]
         timestamp for which to request the data
-    
+
+    zkey : int
+        Key for altitude definition (2 is default)
+        1: xz is the radial distance from the center of the planet (km).
+        2: xz is the altitude above the Martian zero datum (Mars geoid or “areoid”) (km).
+        3: xz is the altitude above the local surface (km).
+        4: xz is the pressure level (kPa).
+        5: xz is the altitude above reference radius (3,396.106 km) (km).    
+
     Returns
     -------
     pandas.Series
         atmoshperic density (kg/m^3)
         index = altitude above MOLA_0 (km)
     """
+
+    # Define a dictionary of altitude types (2 is default)
+    altitude_type = {1: "radial distance from the center of the planet (km)",
+                     2: "altitude above MOLA_0 (km)",
+                     3: "altitude above the local surface (km)",
+                     4: "pressure level (kPa)",
+                     5: "altitude above reference radius (km)"}
+    
     latitude = _check_number(latitude, "latitude", True, -90, True, 90, True)
     longitude = _check_number(longitude, "longitude", True, -180, False, 180, True)
     if not isinstance(timestamp, (datetime.date, datetime.datetime)):
         raise TypeError("timestamp must be a date or datetime object")
     
-    url = _request_url(latitude, longitude, timestamp)
+    url = _request_url(latitude, longitude, timestamp, zkey)
     txt_url = _get_txt_url(url)
     dataframe = _load_and_parse_txt_file(txt_url)
-    
-    dataframe.index *= 1e-3
-    dataframe.index.name = "altitude above MOLA_0 (km)"
+
+    # MCD provides results in m (or Pa); convert to km (kPa) and
+    # set appropriate index name
+    dataframe.index *= 1e-3 
+    dataframe.index.name = altitude_type[zkey]
     
     return dataframe.iloc[:, 0]
 
 
 ###################################################
-def _request_url(latitude, longitude, timestamp):
+def _request_url(latitude, longitude, timestamp, zkey):
     """Converts latitude, longitude and timestamp into a request url to the website"""
     
     jdate = sum(jdcal.gcal2jd(timestamp.year, timestamp.month, timestamp.day))
 
     if isinstance(timestamp, datetime.datetime):
         jdate += timestamp.hour / 24 + timestamp.minute / (24*60) + timestamp.second / (24*3600)
     
-    url = BASE_URL + "cgi-bin/mcdcgi.py?"
+    url = BASE_URL + "cgi-bin/mcdcgi.py?&datekeyhtml=0&localtime=0"
```

Comment:
What are these two new parameters doing?

Reply:
I think these are necessary so that `julian` is correctly interpreted. Without these parameters, the date was being incorrectly interpreted (but I am not sure how). See http://www-mars.lmd.jussieu.fr/mars/info_web/user_manual_5.3.pdf